ledger-service/http-json-oracle/src/it/resources/logback.xml

```diff
@@ -13,7 +13,10 @@
     <logger name="io.grpc.netty" level="WARN">
         <appender-ref ref="STDOUT"/>
     </logger>
+    <logger name="com.daml.http.dbbackend" level="DEBUG">
+        <appender-ref ref="stderr-appender"/>
+    </logger>
```

Comment:
Please remove this extra logging too.

Reply:
d289dc7ebdcd96e0bc4c15baa0b183a3640272f8